Sources/Kommunicate/Classes/Utilities/KMCustomisation.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  KMCustomisation.swift
+//  Kommunicate
+//
+//  Created by Aman on 14/02/23.
+//
+
+import Foundation
+import KommunicateChatUI_iOS_SDK
+
+public class KMCustomization {
```

Comment:
We already have similar class names..Change the name like KMConfigurationSetter, KMConfigurationGenerator etc.. it should not be accessible from outside